projects/documentation/content/migrations/2023-6-11.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: guide.njk
+title: 'Migration Guide: Spectrum Web Components'
+displayName: 2023/6/11
+slug: migration-guide
+---
+
+## Migration: 2023/6/11
+
+As of the 0.33.0 release of the Spectrum Web Components library, we will be leveraging a new version of our Overlay API. We've done our best to ensure a smooth transition from one version of the API to the next, including adding extended support for the argument signature from the previous version. In this way, consumption of elements from the library (e.g. `<overlay-trigger>`, `<sp-picker>`, `<sp-tooltip>`, et al.) or the imperative Overlay API (e.g. `Overlay.open()`) should continue to afford as close to the same functionality as the provided you in the past. Under the covers, many important changes have been made and there are several things you can do in preparation for those changes in your application's lifecycle.
```

Comment:
As of the 0.33.0 release of the Spectrum Web Components library, we will be leveraging a new version of our Overlay API. We've done our best to ensure a smooth transition from one version of the API to the next, including adding extended support for the argument signature from the previous version. In this way, consumption of elements from the library (e.g. `<overlay-trigger>`, `<sp-picker>`, `<sp-tooltip>`, et al.) or the imperative Overlay API (e.g. `Overlay.open()`) should continue affording the closest functionality as what was provided to you in the past. Under the covers, many important changes have been made and there are several things you can do to prepare within your application's lifecycle.